build.gradle

```diff
@@ -22,6 +22,7 @@ plugins {
 
 allprojects {
     repositories {
+        mavenLocal()
```

Comment:
Obviously this and the a-c change will be removed once I'm no longer needing to test on a local build.